rash_book/src/vars.md

```diff
@@ -1,7 +1,7 @@
 # Vars <!-- omit in toc -->
 
 The `rash` context has variables associated to use as [Tera](https://tera.netlify.app/) templates.
-You can use them everywhere except in Yaml keys. They will be rendered at execution time.
+You can use them everywhere except in Yaml keys. Rash will render them at execution time.
```

Comment:
I have a problem, I prefer use `rash` to reference our sotfware and for headers I think it is better Rash but I'm not full consistent i this behaviour.

Reply:
No problem! However, I think we should highlight it somehow. Particularly here, since it's the beginning of a sentence. What if we print it as code, just as you did in line 3?

Reply:
ye, I mean, always as code `\`rash\``

Reply:
Oh yes, I saw that, but I thought you simply meant using _rash_, without the capital letter 😅 It's fixed anyway!